lcUILua/createActions/circleoperations.lua

```diff
@@ -1,5 +1,8 @@
 CircleOperations = {}
 CircleOperations.__index = CircleOperations
+local Circle_FirstPoint
+local Circle_SecondPoint
+local Circle_ThirdPoint
```

Comment:
You should move those 3 variables in the `_init` function like this:
`self.firstPoint = nil`

Those variables are currently shared between the CircleOperations instances.
You can have conflicts if you have multiple windows open creating circles.

Reply:
Ok. I will make the corrections.